src/commands/feedback.js

```diff
@@ -4,7 +4,7 @@ const { Command } = require('@oclif/command');
 
 class Feedback extends Command {
   async run() {
-    const link = chalk.bold('>>>>>  ') + chalk.underline.bold('https://bit.ly/twilio-cli-feedback') + chalk.bold('  <<<<<');
+    const link = chalk.bold('>>>>>  ') + chalk.underline.bold('https://twil.io/twilio-cli-feedback') + chalk.bold('  <<<<<');
```

Comment:
The link is 1 char longer so need to drop a ` `.
```
***********************************************************************
* Ahoy! Thank you for using twilio-cli!                               *
* We are always striving to make twilio-cli better.                   *
* Visit the link below to share your feedback with our team.          *
*                                                                     *
*          >>>>>  https://twil.io/twilio-cli-feedback  <<<<<           *
***********************************************************************
```

This also needs updating: https://github.com/twilio/plugin-debugger/blob/master/src/commands/debugger/logs/list.js#L95